src/Symfony/Component/Validator/Tests/Constraints/AbstractComparisonValidatorTestCase.php

```diff
@@ -16,26 +16,6 @@
 use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
 use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
 
-class ComparisonTest_Class
```

Comment:
ok, i see why you added the new class on top as well in `BicValidatorTest` :)

i still think we prefer classes at the bottom, but technically this change applies to 3.4 and is not really related to your PR.

Reply:
@ro0NL should I remove my changes to these files then? and someone else makes another PR for this?

Reply:
i think we should avoid this conflict with lower (maintained) branches yes